### DISCUSSION TITLE
3D stratified turbulence

### DISCUSSION THREAD

**Discussion by arunkv7** [FourierFlows/GeophysicalFlows.jl#324] [General]
Hi ,
I want to write 3d stratified Navier-Stokes turbulence with GeophysicalFlows.jl. Is that any straight-forward? Any comments in this regard will be helpful. The equations that I want to code are Boussinesq equations with Re, Fr and Pr as parameters. (No rotation)

ps: I have experience in working with Burgers in python (and using FourierFlows.jl)?

Thanks, 
Arun

**Comment by navidcy** [comment]
That can happen for sure. Will there be periodic boundary conditions in `z`?
How straight-forward is relative actually :) I don't know how to quantify this.

A simple example module is [here](https://fourierflows.github.io/FourierFlowsDocumentation/dev/literated/OneDShallowWaterGeostrophicAdjustment/) and also all other modules in the package.
